### ISSUE TITLE
Compare VICR WSD_Parks boundary to Lands

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#510]
Currently pulling from WSD_Parks, which is noticeably different than the latest Lands release from 20160107:

![vicr](https://cloud.githubusercontent.com/assets/1238636/12958196/bece6ae0-cfed-11e5-8556-3e7e9b58c740.jpg)

Will confirm with Chris @ SER.

**Comment by chadlawlis** [comment]
Done via:

``` sql
-- update geom_poly from WSD_Parks to lands_20160107
UPDATE parks_poly
SET geom_poly = (SELECT wkb_geometry FROM lands_20160107 WHERE unit_code = 'VICR')
WHERE unit_code = 'vicr';

-- update unit_area from geometry, data_source to Lands
UPDATE parks
SET unit_area = (SELECT ST_Area(geom_poly) FROM parks_poly WHERE unit_code = 'vicr'),
data_source = 'Land Resources Division'
WHERE unit_code = 'vicr';
```

**Comment by chadlawlis** [comment]
Reopening - unfortunately the Lands update was generated incorrectly.

From Christy @ VIIS, 2/12/16:

> On what you show the south shore in the purple is correct. 
> That area is two separate polygons The pink on the south shore is incorrect NPS does not own the shoreline or submerged resources in that pick area we call the wedge.
> On the North side of the island the latest Landsnet is correct, that should be ALL one piece we do own that shoreline and the submerged resources there.
> 
> Both areas are very critical as they are no take zones that the LE have to patrol. If our maps say we do not own something they can't enforce that regulation.

![image](https://cloud.githubusercontent.com/assets/1238636/13016549/ad0fdc2c-d17c-11e5-9c5c-30b507939eba.png)

So, we need to maintain the northern section as Lands has it and revert the southern section to WSD_Parks, creating a custom VICR source for this unit.

**Comment by chadlawlis** [comment]
From Roger @ Lands:

> Below is a screen capture of what Lands is currently posting for VICR.  There is no break in the northern polygon in either the boundary layer or the ownership data.  I don't know where the wedge in the northern polygon came from.  The southern boundary polygon includes tracts 101-02 (pink wedge) as it is part of the legislated boundary.  Only by including Lands ownership data can users understand that tract 101-02, while within the legislated boundary, is owned by the Government of the Virgin Islands.  Tony please correct me if I misspoke on anything.

<img src="https://cloud.githubusercontent.com/assets/1238636/13156399/8a2d95e2-d63f-11e5-9473-b36588297000.png" width="500px">

In response, from Tony - Chief Cartographer @ SERO:

> You are correct Roger.

Waiting on confirmation from Christy at VIIS before making the transition.

**Comment by chadlawlis** [comment]
Got the go ahead from Christy, good to go for transition to Lands.

**Comment by chadlawlis** [comment]
Done via:

``` sql
-- update boundary geometry from Lands table
UPDATE parks_poly
SET geom_poly = (SELECT wkb_geometry FROM lands_20160107 WHERE unit_code = 'VICR')
WHERE unit_code = 'vicr';

-- update parks table attributes via new boundary
UPDATE parks
SET unit_area = (SELECT ST_Area(geom_poly) FROM parks_poly WHERE unit_code = 'vicr'),
data_source = 'Land Resources Division'
WHERE unit_code = 'vicr';
```

**Comment by chadlawlis** [comment]
Reopening given further feedback from Christy @ VICR:

> With the way it is drawn and the way it shows up on the park tiles, we are going to have issues of confusion with existing and new law enforcement staff as well as local fishermen. Currently the wedge on the south eastern boundary is territory, at least that is my understanding as we never swapped the middle section for that wedge. Therefore it is legal for any kind of fishing or take in that area. The way the map is being displayed by the park now it will be interpreted as that whole southern section of St. John is a no-take and no anchoring area. This will cause an uproar with local fishermen and boaters!.
> 
> When was there a boundary adjustment to include that south east section? 
> 
> Correct me if I am wrong but my understand is that the boundaries you are showing are legislative boundaries? 
> 
> Could the tile show the two areas in question on the south shore in some kind of shaded pattern as you have it in map pasted below? That would help. 

And one more round of feedback:

> is it possible to show the monument the way it is show it the last attachment where it highlights the sections that are not ours? The areas shaded in brown or bluish are not ours! IF we show them as ours the locals are going to yell.

I have deferred to Tony @ SER and Roger @ Lands to answer the Lands-related questions (when the boundary was adjusted, if it is legislative). I have followed up with Christy to inform her we are looking into the possibility of custom boundary styling to address this issue (cc @mtaylorlong), regardless she needs to send us a custom boundary if she would like to opt out of what Lands provides.
